src/MemoryTracing/MemoryTracingUtils.h

```diff
@@ -12,17 +12,24 @@
 
 namespace orbit_memory_tracing {
 
+[[nodiscard]] orbit_grpc_protos::SystemMemoryUsage CreateAndInitializeSystemMemoryUsage();
+// All the following GetValues methods only update the input memory proto when the input file
```

Comment:
I'm ok with this (as discussed online) but I would like to have an additional opinion from @ronaldfw

Reply:
I think it's fine to not update the proto behind the pointer in case there is a failure in the method. I would consider returning an error message in case there is a problem. Or is there a reason you are not returning an error?

Reply:
Thanks for the suggestions. I changed to return an `ErrorMessage` in case there is a problem in extracting information from the file content.